src/_locales/en/messages.json

```
@@ -80,6 +80,10 @@
     "message": "Permission request failed.",
     "description": "Error shown when a permission request failed. This indicates a technical error and does NOT mean the user declined the permission."
   },
+  "errorQrCodeOverflow": {
+    "message": "Cannot generate QR code, the provided text is too much.",
```

I know, I am correcting myself, but as per https://design.firefox.com/photon/copy/writing-for-users.html#be-human I'd rather choose something that sounds better than trying to be 100% technically correct.

I.e.:
```suggestion
    "message": "Cannot generate QR code, the provided text is too long.",
```

("much text" sounds awkward IMHO)

---

Any comment here? You did not include this change...